### DISCUSSION TITLE
[solved] Show editor placeholder on tw.com

### DISCUSSION THREAD

**Discussion by twMat** [TiddlyWiki/TiddlyWiki5#5367] [Ideas]
Edit: Don't waste your time on this thread. It was based on a mistake on my part.

-----------------------------

The default editor on tiddlywiki.com looks like so:

![image](https://user-images.githubusercontent.com/4534056/103575116-77349c00-4ed1-11eb-9384-2c5243622ac9.png)

but in empty.html it looks like so:

![image](https://user-images.githubusercontent.com/4534056/103575144-86b3e500-4ed1-11eb-9961-ce2bb5f5dd4e.png)

IMO the more important place to show the clarifying placeholder text is in the *demo*, i.e on tiddlywiki.com. That is likely where people first are exposed to the edit view. (Once you've seen it, you probably never forget it so empty.htm is probably less important.)

I.e, I propose there should be the placeholder in the demo!

**Comment by pmario** [comment]
@twMat .. Did you have a look at tiddlywiki.com? It looks exactly as you want

**Reply by pmario** [reply]
The "prerelease" looks different, since it uses the codemirror-editor

**Reply by twMat** [reply]
Ahaaa! I am mistaken.
Thank you @pmario !
